packages/cli/src/scaling/scaling.service.ts

```diff
@@ -86,7 +86,9 @@ export class ScalingService {
 	}
 
 	async findJobsByStatus(statuses: JobStatus[]) {
-		return await this.queue.getJobs(statuses);
+		const jobs = await this.queue.getJobs(statuses);
+
+		return jobs.filter((job) => job !== null);
```

Comment:
In what case can the `Job` be null? A comment would be in order since according to the function types it should never return a null